zeppelin-web-vue/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>Zeppelin Studio</title>
```

Comment:
Please rename it still to `Zeppelin Notebook`, changing product name is beyond the scope of this PR.

Reply:
Will fix it.

Reply:
Done.